### PR TITLE
Fixes chakra external links

### PR DESCRIPTION
**Pull request by Royal-lobster**
Fixes https://github.com/EveripediaNetwork/issues/issues/825

### CODE REVIEW

File: src/components/Profile/UserSocialLinks.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const UserSocialLinks = ({ links, address }: ProfileLinksProps) => {
           return (
             <Link
               position="relative"
-              rel="nofollow"
+              isExternal
```
Comment (kesar) [inline src/components/Profile/UserSocialLinks.tsx:26]: i think isExternal only adds the noreferrer but we need the nofollow

Comment (Royal-lobster) [reply src/components/Profile/UserSocialLinks.tsx:26]: Oh let me investigate this further, i guess chakra ui is not giving us the option to add nofollow, i think we can create our own chakra.a link possibly without using the import of `@chakra/system`

Comment (Royal-lobster) [reply src/components/Profile/UserSocialLinks.tsx:26]: @kesar found it, its link overlay the issue - https://github.com/chakra-ui/chakra-ui/blob/95aa4499c72de28e91dd9274177c529db55e7964/packages/components/layout/src/link-box.tsx#L16 here the rest of the props are spread before the rel so it is getting overriden

Comment (kesar) [reply src/components/Profile/UserSocialLinks.tsx:26]: makes sense 👍🏻

**Review by kesar — APPROVED** [review]

**Pull request merged**